### PR TITLE
Added file for footer

### PR DESCRIPTION
**Pull request by kgeorgiev77**

### CODE REVIEW

File: footer.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (Neychok) [inline footer.php:2]: We can't have <html> and <!DOCTYPE html> in the footer. Can we remove this?

File: footer.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<body>
```
Comment (Neychok) [inline footer.php:4]: Why do we open the body tag inside the footer? This is wrong

**Review by Neychok — CHANGES_REQUESTED** [review]
Yeah, I can't approve this. The HTML in the `footer.php` is wrong and has to be corrected.

You can actually see which code `footer.php` replaces -> https://i.imgur.com/4mKghSh.png
So why do we have all the other code there?

File: view-submission.php [hunk 3]
```diff
@@ -66,9 +66,11 @@
 			</section>	
 		</main>
 
-		<?php
-			include 'footer.php';
-		?>
+		<div class="site-wrapper">
```
Comment (Neychok) [inline view-submission.php:69]: Let's not wrap the footer with site-wrapper at all.

**Review by Neychok — CHANGES_REQUESTED** [review]

File: view-submission.php (hunk 3 above)
Comment (kgeorgiev77) [reply view-submission.php]: It's done. I committed the changes.

**Review by Neychok — APPROVED** [review]
Looks better.

**Pull request merged**